### ISSUE TITLE
Question - two custom filters

### ISSUE THREAD

**Issue by gituporabnik** [uBlock-LLC/uBlock#1483]
I came across 2 rules, here are the examples:

1) <code>ekipa24.si###banner1</code>

2) <code>med.over.net###phorum > .main-container > .clearfix.box-holder > .span-3_2.span > .forum.box-full.box > .post-listing > .main-post.single-post:nth-of-type(5) > .clearfix.oglasnik-addbox</code>

In the first case, I would like to replace "banner1" or "1" with universal character (i.e. with any number).
In the second case, it is similar. I would like to replace number "5" (in this case it is in brackets).

I would like to have a single filter for each rule, if possible. I have tried to replace characters with <code>[id^="-"]</code>
For example: <code>ekipa24.si##[id^="banner-"]</code> and <code>ekipa24.si##banner[id^="-"]</code>

But it doesn't work.

**Comment by AlexVallat** [comment]
Is this really Firefox-specific? It seems unlikely.

**Comment by ghost** [comment]
- `ekipa24.si##[id^="banner"]` https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors
- second is more complicated: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type tester: http://lea.verou.me/demos/nth.html

**Comment by gituporabnik** [comment]
@AlexVallat it is most likely not Firefox specific. I changed the title.

However, I figured it out. In the second rule, the number (in this case "5") can be simply replaced with "n", since it can be any character. Therefore:
<code>med.over.net###phorum > .main-container > .clearfix.box-holder > .span-3_2.span > .forum.box-full.box > .post-listing > .main-post.single-post:nth-of-type(n) > .clearfix.oglasnik-addbox</code>

Thank you @gwarser for suggestions! The first rule you provided also works well.
